### PR TITLE
refactor(dms/kafka_instance): fix lint issues

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

Fix some lint issues in `huaweicloud_dms_rabbitmq_instance`.

**Release note**:

```release-note
NONE
```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms' TESTARGS='-run TestAccKafkaInstance'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms -v -run TestAccKafkaInstance -timeout 360m -parallel 4
=== RUN   TestAccKafkaInstancesDataSource_basic
--- PASS: TestAccKafkaInstancesDataSource_basic (592.66s)
=== RUN   TestAccKafkaInstance_basic
=== PAUSE TestAccKafkaInstance_basic
=== RUN   TestAccKafkaInstance_withEpsId
=== PAUSE TestAccKafkaInstance_withEpsId
=== RUN   TestAccKafkaInstance_compatible
=== PAUSE TestAccKafkaInstance_compatible
=== RUN   TestAccKafkaInstance_newFormat
=== PAUSE TestAccKafkaInstance_newFormat
=== CONT  TestAccKafkaInstance_basic
=== CONT  TestAccKafkaInstance_compatible
=== CONT  TestAccKafkaInstance_withEpsId
=== CONT  TestAccKafkaInstance_newFormat
--- PASS: TestAccKafkaInstance_withEpsId (585.26s)
--- PASS: TestAccKafkaInstance_compatible (998.31s)
--- PASS: TestAccKafkaInstance_newFormat (2309.55s)
--- PASS: TestAccKafkaInstance_basic (2700.08s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       3292.841s
```

### CODE REVIEW

File: huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go [hunk 1]
```diff
@@ -190,6 +190,11 @@ func DataSourceDmsKafkaInstances() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"management_connect_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						// Typo, it is only kept in the code, will not be shown in the docs.
 						"manegement_connect_address": {
```
Comment (houpeng80) [inline huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go]: it is better to mark it deprecated

File: huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go [hunk 2]
```diff
@@ -226,6 +225,11 @@ func ResourceDmsKafkaInstance() *schema.Resource {
...
 				Computed: true,
 			},
+			"management_connect_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			// Typo, it is only kept in the code, will not be shown in the docs.
 			"manegement_connect_address": {
 				Type:     schema.TypeString,
 				Computed: true,
```
Comment (houpeng80) [inline huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go]: ditto

File: huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go [hunk 3]
```diff
@@ -258,6 +262,11 @@ func ResourceDmsKafkaInstance() *schema.Resource {
...
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"listener_ip": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						// Typo, it is only kept in the code, will not be shown in the docs.
 						"lisenter_ip": {
 							Type:     schema.TypeString,
 							Computed: true,
```
Comment (houpeng80) [inline huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go]: ditto

File: huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go (hunk 1 above)
Comment (chengxiangdong) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go]: done

File: huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go (hunk 3 above)
Comment (chengxiangdong) [reply huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go]: done

File: huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go (hunk 2 above)
Comment (chengxiangdong) [reply huaweicloud/services/dms/resource_huaweicloud_dms_kafka_instance.go]: done

**Comment by chengxiangdong** [comment]
@houpeng80 
All changes have been pushed, please review

File: huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go (hunk 1 above)
Comment (ShiChangkuo) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go]: it seems that `Deprecated` is still missing, please check it again.

Comment (chengxiangdong) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go]: fixed now

**Pull request merged**